tools/ci/illegal_dme_files.py

```diff
@@ -0,0 +1,42 @@
+# Naive search for illegal files. Has no semantic knowledge, just a lexical
+# search for specific endings in a .dme
+#
+# This is basically a slightly edited verison of unticked_files.py.
+# Look there for precise documentation on the methods used here.
+from pathlib import Path, PureWindowsPath
+import argparse
+import sys
+
+INCLUDED_FILES = [
+    'paradise.dme'
+]
+
+ILLEGAL_FILES = (
+    '.dmm'
+)
+
+def get_illegal_files(root:Path):
+    illegal_files = set()
+    for includer in INCLUDED_FILES:
+        with open(root / includer, 'r') as f:
+            # I've tried to optimize this, but this was the best I could get. Try placing the strips elsewhere if you dare
+            lines = [line for line in f.readlines() if line.rstrip('\r\n').strip('"').endswith(ILLEGAL_FILES)]
+            included = [line.replace('#include ', '').rstrip('\r\n').strip('"') for line in lines]
+            print(f'Found {len(included)} illegal files in {root / includer}')
```

Comment:
Are you sure you need to print this off twice?

Reply:
One prints it once for each file, and one for all of them. For now this just includes 1 files, but I can comment out this print if needed

Reply:
Made it print out the file name if theres more than one file, that should hopefully work.